### PR TITLE
[HomePage] Fixing bug and add feature to invitation card

### PR DESCRIPTION
**Pull request by RenaufalAlrafid**
image

Minta tolong direview gan 
ini masi dalam benetuk component belum terintegrate dengan slidernya

### CODE REVIEW

File: lib/Scenes/Page/HomePage/Component/InvitationCard/Component/invitation_card_page.dart [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                    padding: const EdgeInsets.fromLTRB(0, 2, 8, 2),
+                    child: Text(
+                      widget.invitationCard.taskItem.date,
+                      style: TimeBoxingTextStyle.paragraph3(
+                          TimeBoxingFontWeight.bold,
+                          TimeBoxingColors.neutralBlack()),
+                    ),
+                  ),
+                  Container(
+                    margin: const EdgeInsets.fromLTRB(0, 7, 8, 7),
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/InvitationCard/Component/invitation_card_page.dart]: jangan ganjil ya mas naufal

File: lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart [hunk 2]
```diff
@@ -210,12 +242,97 @@ class _TimeboxingInvitationCardState extends State<TimeboxingInvitationCard> {
...
+                    height: 16,
+                  ),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      Center(
+                          child: Row(
+                        children: [
+                          Container(
+                            margin: const EdgeInsets.fromLTRB(0, 7, 4, 7),
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart]: jangan ganjil ya mas naufal

File: lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart [hunk 3]
```diff
@@ -210,12 +242,97 @@ class _TimeboxingInvitationCardState extends State<TimeboxingInvitationCard> {
...
+                            width: 4,
+                            height: 4,
+                            decoration: BoxDecoration(
+                                color: TimeBoxingColors.primary70(
+                                    TimeBoxingColorType.shade),
+                                borderRadius:
+                                    const BorderRadius.all(Radius.circular(4))),
+                          ),
+                          Container(
+                            margin: const EdgeInsets.fromLTRB(0, 7, 4, 7),
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart]: jangan ganjil ya mas naufal

File: lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart [hunk 4]
```diff
@@ -210,12 +242,97 @@ class _TimeboxingInvitationCardState extends State<TimeboxingInvitationCard> {
...
+                            width: 4,
+                            height: 4,
+                            decoration: BoxDecoration(
+                                color: TimeBoxingColors.text70(
+                                    TimeBoxingColorType.tint),
+                                borderRadius:
+                                    const BorderRadius.all(Radius.circular(4))),
+                          ),
+                          Container(
+                            margin: const EdgeInsets.fromLTRB(0, 7, 4, 7),
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart]: jangan ganjil ya mas naufal

File: lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart [hunk 5]
```diff
@@ -210,12 +242,97 @@ class _TimeboxingInvitationCardState extends State<TimeboxingInvitationCard> {
...
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart]: jangan ganjil ya mas naufal

File: lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart [hunk 6]
```diff
@@ -210,12 +242,97 @@ class _TimeboxingInvitationCardState extends State<TimeboxingInvitationCard> {
...
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart]: jangan ganjil ya mas naufal

File: macos/Flutter/GeneratedPluginRegistrant.swift [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+//
+
+import FlutterMacOS
+import Foundation
+
+import firebase_core
+
+func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
+  FLTFirebaseCorePlugin.register(with: registry.registrar(forPlugin: "FLTFirebaseCorePlugin"))
+}
```
Comment (gilangpersada) [inline macos/Flutter/GeneratedPluginRegistrant.swift]: lho lho ndak bahaya tah?

Comment (RenaufalAlrafid) [reply macos/Flutter/GeneratedPluginRegistrant.swift]: bahaya

File: lib/Scenes/Page/HomePage/Component/InvitationCard/Component/invitation_card_page.dart (hunk 1 above)
Comment (RenaufalAlrafid) [reply lib/Scenes/Page/HomePage/Component/InvitationCard/Component/invitation_card_page.dart]: done

File: lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart (hunk 2 above)
Comment (RenaufalAlrafid) [reply lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart]: done

File: lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart (hunk 3 above)
Comment (RenaufalAlrafid) [reply lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart]: done

File: lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart (hunk 4 above)
Comment (RenaufalAlrafid) [reply lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart]: done

File: lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart (hunk 5 above)
Comment (RenaufalAlrafid) [reply lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart]: done

File: lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart (hunk 6 above)
Comment (RenaufalAlrafid) [reply lib/Scenes/Page/HomePage/Component/InvitationCard/invitation_card.dart]: done

**Comment by RenaufalAlrafid** [comment]
image
preview after fixing comment

**Review by gilangpersada — APPROVED** [review]

File: macos/Flutter/GeneratedPluginRegistrant.swift (hunk 7 above)
Comment (gilangpersada) [reply macos/Flutter/GeneratedPluginRegistrant.swift]: resolved

**Review by irfandigalih — APPROVED** [review]

**Pull request merged**